.jenkins/test_config.yaml

```diff
@@ -43,4 +43,15 @@ stages:
         command: cd .jenkins/lm-eval-harness && bash run-tests.sh -c configs/models-mss.txt -t 2
       - name: gsm8k_small_g2_tp2_mss
         flavor: g2.s
-        command: cd .jenkins/lm-eval-harness && bash run-tests.sh -c configs/models-mss.txt -t 2
\ No newline at end of file
+        command: cd .jenkins/lm-eval-harness && bash run-tests.sh -c configs/models-mss.txt -t 2
+  - name: test_gsm8k_spec_decode
+    steps:
+      - name: gsm8k_small_g2_tp1_mlp_spec_decode
+        flavor: g2
+        command: VLLM_CONTIGUOUS_PA=false VLLM_SKIP_WARMUP=True pytest -v tests/spec_decode/e2e/test_mlp_correctness.py::test_mlp_e2e_greedy_correctness 
```

Comment:
How does it work? Will jenkins find the 'test' directory in vllm directory? Should you set it as working directory first?

Reply:
Yes, I assume the jenkins runs in 'vllm-fork' folder, and pytest will find UT based on path.